scenario/scenario.go

```diff
@@ -0,0 +1,81 @@
+package scenario
+
+import (
+	"context"
+	"math/big"
+	"testing"
+
+	"github.com/0xProject/0x-mesh/constants"
+	"github.com/0xProject/0x-mesh/ethereum"
+	"github.com/0xProject/0x-mesh/ethereum/wrappers"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/stretchr/testify/require"
+)
+
+var eighteenDecimalsInBaseUnits = new(big.Int).Exp(big.NewInt(10), big.NewInt(18), nil)
+var wethAmount = new(big.Int).Mul(big.NewInt(50), eighteenDecimalsInBaseUnits)
+var zrxAmount = new(big.Int).Mul(big.NewInt(100), eighteenDecimalsInBaseUnits)
+
+// SetupBalancesAndAllowances sets up the proper balance/allowance for the maker/taker of
+// ZRX/WETH respectively so that the created orders are fillable.
```

Comment:
Can we make `wethAmount` and `zrxAmount` parameters for this function so that it is more generally useful? As written, it's also not clear which orders you're referring to and what it sets the balances and allowances to exactly.